go.mod

```diff
@@ -1,3 +1,8 @@
 module k8s.io/kubeadm
 
 go 1.12
+
+require (
+	github.com/spf13/cobra v0.0.5 // indirect
+	k8s.io/kubeadm/kinder v0.0.0-20191127122120-f4019dfc8ffd // indirect
+)
```

Comment:
thanks for the PR, the root go.mod and go.sum should not be updated.

Reply:
fixed. My bad.